store-implementation/parquet-store/src/main/java/uk/gov/gchq/gaffer/parquetstore/utils/GafferGroupObjectConverter.java

```diff
@@ -133,7 +133,9 @@ public Object sparkRowToGafferObject(final String gafferColumn, final Row row) t
         if (paths[0].contains(".")) {
             final Object nestedRow = row.getAs(gafferColumn);
             if (nestedRow != null) {
-                if (nestedRow instanceof GenericRowWithSchema) {
+                if (nestedRow instanceof String[]) {
```

Comment:
Why is this change necessary? Any arrays that have come from a Spark row should be a WrappedArray.